front/app/api/follow_unfollow/useFollowers.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import followUnfollowKeys from './keys';
+import { FollowUnfollowKeys, IFollowers, IParameters } from './types';
+
+const fetchFollowers = async ({ followableObject }: IParameters) =>
```

Comment:
Would this followableObject be a UUID?

Reply:
Ah ok - sorry, just wondering how this works. So this useFollowers hook gets the followers for a specific object correct? Do we not need to pass in an object ID somewhere? :thinking:

Reply:
Ahh alright, now I understand! Thanks!! :)